fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/AggregateIndexExpansionVisitor.java

```diff
@@ -246,9 +256,11 @@ private Pair<SelectExpression, List<CorrelationIdentifier>> constructSelectHavin
 
         final var placeholderAliases = ImmutableList.<CorrelationIdentifier>builder();
         final var selectHavingGraphExpansionBuilder = GraphExpansion.builder().addQuantifier(groupByQun);
+        final List<Value> groupingValues = groupingValueReference == null ? Collections.emptyList() : Values.deconstructRecord(groupingValueReference);
```

Comment:
During an in-person review of this code, this change came up. To be clear, the change here was to construct the list of grouping values before going through and assigning place holders. That way, the _size_ of the grouping values can be referenced below in the `isPermuted()`-guarded statement.